<ui version="4.0" >
 <class>QPageSetupWidget</class>
 <widget class="QWidget" name="QPageSetupWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>416</width>
    <height>488</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3" >
   <property name="margin" >
    <number>0</number>
   </property>
   <item row="0" column="0" colspan="2" >
    <layout class="QHBoxLayout" name="horizontalLayout_4" >
     <item>
      <widget class="QComboBox" name="unit" />
     </item>
     <item>
      <spacer name="horizontalSpacer_3" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2" >
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="title" >
      <string>Paper</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" >
      <item row="0" column="0" >
       <widget class="QLabel" name="pageSizeLabel" >
        <property name="text" >
         <string>Page size:</string>
        </property>
        <property name="buddy" >
         <cstring>paperSize</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QComboBox" name="paperSize" />
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="widthLabel" >
        <property name="text" >
         <string>Width:</string>
        </property>
        <property name="buddy" >
         <cstring>paperWidth</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <layout class="QHBoxLayout" name="horizontalLayout_3" >
        <item>
         <widget class="QDoubleSpinBox" name="paperWidth" >
          <property name="maximum" >
           <double>9999.989999999999782</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="heightLabel" >
          <property name="text" >
           <string>Height:</string>
          </property>
          <property name="buddy" >
           <cstring>paperHeight</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="paperHeight" >
          <property name="maximum" >
           <double>9999.989999999999782</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="paperSourceLabel" >
        <property name="text" >
         <string>Paper source:</string>
        </property>
        <property name="buddy" >
         <cstring>paperSource</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QComboBox" name="paperSource" />
      </item>
      <item row="1" column="2" >
       <spacer name="horizontalSpacer_4" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QGroupBox" name="groupBox_3" >
     <property name="title" >
      <string>Orientation</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout" >
      <item>
       <widget class="QRadioButton" name="portrait" >
        <property name="text" >
         <string>Portrait</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="landscape" >
        <property name="text" >
         <string>Landscape</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="reverseLandscape" >
        <property name="text" >
         <string>Reverse landscape</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="reversePortrait" >
        <property name="text" >
         <string>Reverse portrait</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_5" >
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item rowspan="2" row="2" column="1" >
    <widget class="QWidget" native="1" name="preview" />
   </item>
   <item row="3" column="0" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Margins</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2" >
      <item>
       <layout class="QGridLayout" name="gridLayout" >
        <item row="0" column="1" >
         <widget class="QDoubleSpinBox" name="topMargin" >
          <property name="toolTip" >
           <string>top margin</string>
          </property>
          <property name="accessibleName" >
           <string>top margin</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="maximum" >
           <double>999.990000000000009</double>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="3" >
         <layout class="QHBoxLayout" name="horizontalLayout" >
          <item>
           <spacer name="horizontalSpacer_7" >
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0" >
             <size>
              <width>0</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="leftMargin" >
            <property name="toolTip" >
             <string>left margin</string>
            </property>
            <property name="accessibleName" >
             <string>left margin</string>
            </property>
            <property name="alignment" >
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="maximum" >
             <double>999.990000000000009</double>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer" >
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType" >
             <enum>QSizePolicy::MinimumExpanding</enum>
            </property>
            <property name="sizeHint" stdset="0" >
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="rightMargin" >
            <property name="toolTip" >
             <string>right margin</string>
            </property>
            <property name="accessibleName" >
             <string>right margin</string>
            </property>
            <property name="alignment" >
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="maximum" >
             <double>999.990000000000009</double>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_8" >
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0" >
             <size>
              <width>0</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="2" column="1" >
         <widget class="QDoubleSpinBox" name="bottomMargin" >
          <property name="toolTip" >
           <string>bottom margin</string>
          </property>
          <property name="accessibleName" >
           <string>bottom margin</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="maximum" >
           <double>999.990000000000009</double>
          </property>
         </widget>
        </item>
        <item row="0" column="2" >
         <spacer name="horizontalSpacer_2" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::MinimumExpanding</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>0</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="0" >
         <spacer name="horizontalSpacer_5" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::MinimumExpanding</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>0</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0" >
    <spacer name="verticalSpacer" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
